Client/Frontend/Browser/Onboarding/Welcome/PrivacyEverywhereView.swift

```diff
@@ -26,29 +26,33 @@ struct PrivacyEverywhereView: View {
             }
             VStack(spacing: 10) {
                 Text(Strings.Callout.privacyEverywhereCalloutTitle)
-                    .font(.title3.weight(.medium))
+                    .font(.system(size: 20, weight: .medium))
                     .foregroundColor(Color(.bravePrimary))
                 Image(uiImage: #imageLiteral(resourceName: "privacy-everywhere-image"))
                     .resizable()
                     .aspectRatio(contentMode: .fit)
                 Text(Strings.Callout.privacyEverywhereCalloutDescription)
```

Comment:
Why are all the dynamic fonts changing to static?

Reply:
Changes at https://github.com/brave/brave-ios/pull/4535/commits/58ccc7f81e7b459fd6be2dca304f4b2500b13868